main.py

```diff
@@ -83,7 +83,8 @@ def find_close_matches(patterns_data, min_matches, threshold):
                 matches_list.matches.append(close_match)
         if len(matches_list.matches) > min_matches:
             all_matches_list.append(matches_list)
-    print(str(len(all_matches_list)) + " melodic intervals had more than " + str(min_matches) + " exact or close matches.\n")
+    print(str(len(all_matches_list)) + " melodic intervals had more than " + str(
```

Comment:
No need to change it, but fyi, print() can handle different types as long as they're comma separated. And a space is the default separator so you don't need to add it. So you could write stuff like:
print(len(all_matches_list), 'melodic intervals had more than', min_matches, 'exact or close matches.\n')
I think that makes stuff a little easier to read.
... oh wait, I see that this is probably one of the automatic changes you mentioned (are you using Prettier?). Yeah, definitely don't worry about this.